packages/frontend-design-poc/src/api/queries.ts

```diff
@@ -1,9 +1,12 @@
 import { GetDialogDtoSO } from 'dialogporten-types-generated';
 
-interface User {
-  name: string;
-  roles: string[];
-}
-
-export const getUser = (): Promise<User> => fetch('/user').then((resp) => resp.json());
+export const getIsAuthenticated = (): Promise<boolean> =>
```

Comment:
Instead of `api/queries.ts` I would suggest putting it in a folder called `auth/` (together with the rest I commented on).